content/upload/upload_2.md

```diff
@@ -0,0 +1,200 @@
++++
+title = "Data upload - Databricks"
+weight = 2
+post = ""
++++
+
+{{% attachments style="blue" %}}
+{{% /attachments %}}
+
+
+You can load the sample data to the warehouse using your `Databricks Workspace` as described in the below steps. Please download the *sample_events.csv* from the attachments. For more details please check out the official [Databricks documentation](https://docs.databricks.com/ingestion/add-data/index.html).
```

Comment:
This might be out of scope for this release, but you can do Delta Sharing within Databricks now - @georgewoodhead did this for the composable CDP accelerator with Databricks. It means the assets are also available for others to use, no imports required! I wonder if we can do the same here, don't block the release for this, but have a chat to George when he's back to see if we could do that here too

Reply:
Thanks, Paul, I'll talk to him about this!